logger/logger.go

```diff
@@ -0,0 +1,52 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/goat-project/goat-one/constants"
+	"github.com/spf13/viper"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
As mentioned in the documentation [https://github.com/sirupsen/logrus#logrus-](https://github.com/sirupsen/logrus#logrus-) the import should use lowercase `sirupsen`.